### ISSUE TITLE
Firewall Section insert_before bottom_line

### ISSUE THREAD

**Issue by pokepoke81** [terraform-providers/terraform-provider-nsxt#208]
Please clarify in the documentation where "bottom_line" comes from in the insert_before rule and if there are any other special rule ID's that can be used.

**Comment by annakhm** [comment]
Hi @pokepoke81,
We have introduced a new (recommeded) way of consuming resources (not yet released). The new resource for firewall section is `nsxt_policy_security_policy`, and it exposes `sequence_number` attribute to control section order.
However, if you prefer to use the old resource - I'd like to understand your need a bit better, since "bottom_line" is a moving target. Are you looking to add a section at the bottom of pre-configured sections (which are not added via the provider)?

**Comment by pokepoke81** [comment]
Security policies sounds much better. My issue was that I couldn't figure out where the "bottom_line" referenced in the examples came from. It appears to be generic and not actually do anything. 

To work around this, I created a list of dummy sections with Terraform in the order I need them, then use them to insert other rules against. I had to find the UUID of the Default Rule so that it would put them in the appropriate order. It's a hack, but got me what I needed.
